packages/backend-common/src/reading/GithubUrlReader.test.ts

```diff
@@ -24,8 +26,19 @@ import {
   ProviderConfig,
   readConfig,
 } from './GithubUrlReader';
+import fs from 'fs';
+import path from 'path';
+import mockfs from 'mock-fs';
+import { resolvePackagePath } from '../';
+import recursive from 'recursive-readdir';
 
 describe('GithubUrlReader', () => {
+  const dirname = resolvePackagePath(
```

Comment:
Isn't `__dirname` ok to use in tests? @Rugvip 

Just thinking that a relative path to a `__fixtures__` folder would be simplest